README.md

```diff
@@ -147,6 +148,12 @@ This class extends [`EventEmitter`][] from Node.js.
     handling I/O in parallel.
   * `useAtomics`: (`boolean`) Use the [`Atomics`][] API for faster communication
     between threads. This is on by default.
+  * `maxOldGenerationSizeMb`: (`number`) The maximum size of each worker threads
+    main heap in MB.
+  * `maxYoungGenerationSizeMb`: (`number`) The maximum size of a heap space for
+    recently created objects.
+  * `codeRangeSizeMb`: (`number`) The size of a pre-allocated memory range used
+    for generated code.
```

Comment:
We can for sure. Although, my TypeScript-foo is weak and I'm not entirely sure how to export the `ResourceLimits` interface without it being breaking since we use `export = Piscina` in index.ts.

Reply:
We don’t need to export `ResourceLimits` here at all :)

Reply:
Let me know if you’d push that change here, though

Reply:
wait... think I got it figured out that we don't have to.... been way too long since I've mucked around with TypeScript

Reply:
Nope, got it figured out :-D